src/components/search-location/component.jsx

```diff
@@ -87,9 +86,11 @@ function SearchLocation({
     waitingInteraction,
   });
 
-  useEffect(() => {
```

Comment:
For some reason the autofocus renderer is preventing the search result list from appearing. Subsequent renders are not working,